chia/full_node/full_node.py

```diff
@@ -704,7 +710,7 @@ async def sync_from_fork_point(self, fork_point_height: int, target_peak_sb_heig
                     continue
                 elif isinstance(response, RespondBlocks):
                     success, advanced_peak, _ = await self.receive_block_batch(
-                        response.blocks, peer, None if advanced_peak else uint32(fork_point_height)
+                        response.blocks, peer, None if advanced_peak else uint32(fork_point_height), summaries
                     )
                     if success is False:
```

Comment:
We should ban the peer if it's not successful, maybe 5 or 10 minutes at least

Reply:
Ok, i can add that as part of the pr